articles/azure-monitor/platform/powershell-workspace-configuration.md

```diff
@@ -181,6 +181,21 @@ New-AzureRmOperationalInsightsWindowsPerformanceCounterDataSource -ResourceGroup
 New-AzureRmOperationalInsightsCustomLogDataSource -ResourceGroupName $ResourceGroup -WorkspaceName $WorkspaceName -CustomLogRawJson "$CustomLog" -Name "Example Custom Log Collection"
 
 ```
+In the above example regexDelimiter was defined as "\\n" for newline.  The following delimeters can also be used:
```

Comment:
"can also be used" - more accurate to say that those are the supported formats. Using other formats will fail the command.

Reply:
@yoramsinger I changed that line to to reflect your advice.